third_party/ruby_externals.bzl

```diff
@@ -59,6 +59,7 @@ def register_ruby_dependencies():
         strip_prefix = strip_prefix,
         build_file = ruby_build,
         patches = [
+            "@com_stripe_ruby_typer//third_party/ruby:thp.patch",
```

Comment:
Is it important that you put the patch first? We have traditionally put all new patches at the bottom.

Reply:
ack I'll put it at the bottom